jsonschema-core.xml

```diff
@@ -1830,6 +1830,48 @@
                 </section>
             </section>
 
+            <section title="Compound Documents">
+                <t>
+                    A Compound Schema Document is defined as JSON document (sometimes called a "dereferenced" or
+                    "bundled" schema) which has multiple embedded JSON Schema Resources bundled into the same
+                    document to ease transportation.
+                </t>
+                <t>
+                    Each embedded Schema Resource MUST be treated as an individual Schema Resource, following standard
+                    schema loading and processing requirements, including determining voacbulary support.
+                </t>
+                <section title="Bundling">
+                    <t>
+                        A Compound Schema Document is created by replacing references (such as "$ref") with an
+                        embedded version of the referenced Schema Resource.
+                    </t>
+                    <t>
+                        Each embedded JSON Schema Resource (and the document's root) MUST identify itself
+                        with an absolute URI using the "$id" keyword, and SHOULD make use of the "$schema"
+                        keyword to identify the dialect it is using, in the root of the schema resource.
```

Comment:
Do we have wording around how an embedded resource defaults to the `$schema` of the enclosing resource if not specified?  I vaguely recall adding that somewhere but maybe it belongs here now?

Reply:
Maybe. If so it does belong here.

Reply:
It doesn't seem we do. I'll mark this is resolved.

Reply:
It doesn't seem we do. I'll mark this is resolved.